keps/sig-apps/3850-backoff-limits-per-index-for-indexed-jobs/README.md

```diff
@@ -992,6 +990,8 @@ Recall that end users cannot usually observe component logs or access metrics.
 
 ###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
 
+This feature does not propose SLOs.
```

Comment:
> Are there any tests for feature enablement/disablement?

Have those been added? Can you link? If not, can you please prioritize?


> How can a rollout or rollback fail? Can it impact already running workloads?

it doesn't impact already running pods, but it actually impacts the way they are restarted in case of failures - seems worth clarifying.

also, please address "how can rollout/rollback fail"


> Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?

Please ensure that this will happen and the KEP will get updated before graduating FG in k/k.
Even better would be to describe a test scenario now, see
https://github.com/kubernetes/enhancements/pull/3658/files
as an example


> Are there any missing metrics that would be useful to have to improve observability of this feature?

Are you still planning it? Can you update?


 https://github.com/kubernetes/enhancements/tree/master/keps/NNNN-kep-template#can-enabling--using-this-feature-result-in-resource-exhaustion-of-some-node-resources-pids-sockets-inodes-etc

as a new question was added - please copy and fill-in

Reply:
I agree with Wojtek here, this feature is proposed to be promoted to beta, ie. on by default, so all the questions around rollback and monitoring when such should be performed are very important.

Reply:
@wojtek-t @soltysh thanks for reviewing. I think all the questions are now addressed. PTAL.